src/plugins/controls/public/options_list/components/options_list_popover_footer.tsx

```diff
@@ -53,22 +59,39 @@ export const OptionsListPopoverFooter = ({ isLoading }: { isLoading: boolean })
             />
           </div>
         )}
-        <div
+
+        <EuiFlexGroup
+          gutterSize="xs"
+          responsive={false}
+          alignItems="center"
           css={css`
             padding: ${useEuiPaddingSize('s')};
           `}
+          justifyContent={'spaceBetween'}
         >
-          <EuiButtonGroup
-            legend={OptionsListStrings.popover.getIncludeExcludeLegend()}
-            options={aggregationToggleButtons}
-            idSelected={exclude ? 'optionsList__excludeResults' : 'optionsList__includeResults'}
-            onChange={(optionId) =>
-              optionsList.dispatch.setExclude(optionId === 'optionsList__excludeResults')
-            }
-            buttonSize="compressed"
-            data-test-subj="optionsList__includeExcludeButtonGroup"
-          />
-        </div>
+          <EuiFlexItem grow={false}>
+            <EuiButtonGroup
+              legend={OptionsListStrings.popover.getIncludeExcludeLegend()}
+              options={aggregationToggleButtons}
+              idSelected={exclude ? 'optionsList__excludeResults' : 'optionsList__includeResults'}
+              onChange={(optionId) =>
+                optionsList.dispatch.setExclude(optionId === 'optionsList__excludeResults')
+              }
+              buttonSize="compressed"
+              data-test-subj="optionsList__includeExcludeButtonGroup"
+            />
+          </EuiFlexItem>
+          {!allowExpensiveQueries && (
+            <EuiFlexItem data-test-subj="optionsList-allow-expensive-queries-warning" grow={false}>
+              <EuiIconTip
+                type="warning"
+                color="warning"
+                content={OptionsListStrings.popover.getAllowExpensiveQueriesWarning()}
+                aria-label={OptionsListStrings.popover.getAllowExpensiveQueriesWarning()}
+              />
+            </EuiFlexItem>
+          )}
```

Comment:
The popover title used to contain the warning that shows up when the `allowExpensiveQueries` setting is `false` - however, since I removed this title, I moved it to the footer instead: